.github/workflows/common.yml

```diff
@@ -60,13 +61,6 @@ jobs:
     needs: get_r_version
     with:
       r-version: "${{ needs.get_r_version.outputs.r-version }}"
-  readme:
```

Comment:
@rossfarrugia Looks like the template mightve been outdated, I just copy and pasted the same one over from admiral as it is today

Reply:
thanks @zdz2101 - good spot! appreciate if you could make an issue in admiraltemplate repo for this and we can update it later when someone gets chance